api/v3/openapi.yaml

```diff
@@ -1160,6 +1160,38 @@ paths:
         '5XX':
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /clusters/{cluster_id}/topics/{topic_name}/partitions/{partition_id}/offset:
```

Comment:
The api path `offset_type` as well, add that to the path. Also document default behaviour without any `offset_type` specified in the path.